### PR TITLE
Example project enhancements

### PR DESCRIPTION
**Pull request by Iliailineya**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@ target/
 !.mvn/wrapper/maven-wrapper.jar
 !**/src/main/**/target/
 !**/src/test/**/target/
+src/main/resources/app.properties
```
Comment (YeaaaaH) [inline .gitignore]: ?

File: src/main/java/spring/controller/ProjectRestController.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    private final ProjectService projectService;
+
+    public ProjectRestController(ProjectService projectService) {
+        this.projectService = projectService;
+    }
+
+    @PostMapping("/post")
+    public ResponseEntity<Project> createProject(@RequestBody Project project) {
+        log.info("Creating project: " + project);
```
Comment (YeaaaaH) [inline src/main/java/spring/controller/ProjectRestController.java]: too many logs is also not a good practice

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
remove redundant logging

**Comment by Iliailineya** [comment]
Fixed

**Review by YeaaaaH — APPROVED** [review]

**Pull request merged**